### DISCUSSION TITLE
War and Protection

### DISCUSSION THREAD

**Discussion by RoinujNosde** [RoinujNosde/SimpleClans#129] [Ideas]
Give feedback on the new War and Protection feature

**Comment by ChanceSD** [comment]
I would like wars to be sort of a minigame, like when a clan declares war on another a kill counter would start. The first clan to reach 100 kills (or whatever value defined in the config) on the other clan would win that war. Then there could be a clan ranking for most wars won or just add the war victories to the current clan list. Just an idea.

**Reply by RoinujNosde** [reply]
Interesting.
I'm not sure if I am going to implement this idea in SimpleClans, but it could fit in my other plugin (TitansBattle).

**Reply by ChanceSD** [reply]
Oh nice, didn't know about that plugin, I will probably use it since it seems very interesting. I still think these wars could fit on SimpleClans though, the players could even fight without an arena, just in the regular survival world.

**Comment by RoinujNosde** [comment]
Add support for WorldGuard 6
